src/services/todo.php

```diff
@@ -0,0 +1,82 @@
+<?php
+include_once "../database/db.php";
+include_once '../utils/session.php';
+
+function createTask($user_id, $status, $title, $description, 
+                    $created_at, $planned_start_date, 
+                    $planned_end_date, $actual_start_date, 
+                    $actual_end_date) 
+{
+    global $con;
+    $query = "INSERT INTO Tasks
+        (user_id, status, title, description,
+        created_at, planned_start_date, 
+        planned_end_date, actual_start_date, actual_end_date)
+        VALUES ($user_id, $status, $title, $description, 
+        $created_at, $planned_start_date, 
+        $planned_end_date, $actual_start_date, 
+        $actual_end_date);";
+    $result = mysqli_query($con, $query);
+    $user = mysqli_fetch_assoc($result);
+    return $user;
+}
+
+function edit($user_id, $title, $description, 
+            $planned_start_date, $planned_end_date) {
+    global $con;
+    $query = "UPDATE Tasks set title = $title, description = $description, 
+        planned_start_date = $planned_start_date, 
+        planned_end_date = $planned_end_date WHERE id = $user_id;";
+    $result = mysqli_query($con, $query);
+    $user = mysqli_fetch_assoc($result);
+    return $user; 
+}
+
+function deleteTask($task_id) {
+    global $con;
+    $query = "DELETE FROM Tasks WHERE id = $task_id;";
+    $result = mysqli_query($con, $query);
+    $user = mysqli_fetch_assoc($result);
+    return $user;
+}
+
+function updatStatus($user_id,$status) {
+    global $con;
+    $query = "UPDATE Tasks SET status=$status WHERE id=$user_id;";
```

Comment:
We change the task's status based on the task_id not the user_id. Cause there will be no way we are specifying which task we are trying to update